src/common/types/data_chunk.cpp

```diff
@@ -209,11 +209,14 @@ void DataChunk::Print() {
 
 struct DuckDBArrowArrayChildHolder {
 	ArrowArray array;
-	// need max three pointers for strings
+	//! need max three pointers for strings
 	duckdb::array<const void *, 3> buffers = {{nullptr, nullptr, nullptr}};
 	Vector vector = {};
-	unique_ptr<data_t[]> string_offsets = nullptr;
-	unique_ptr<data_t[]> string_data = nullptr;
+	unique_ptr<data_t[]> offsets = nullptr;
```

Comment:
These initializers are not necessary (`nullptr` is the default for unique_ptr, and empty is the default for vector).